exporter/streams_test.go

```diff
@@ -544,3 +544,59 @@ func TestStreamsExtractStreamMetrics(t *testing.T) {
 
 	}
 }
+
+func TestStreamsExtractStreamMetricsExcludeConsumer(t *testing.T) {
```

Comment:
I think this is missing a check for metrics that are *not* present - as in, metrics that normally would be there but due to the `StreamsExcludeConsumerMetrics` check are now not there. Let's add that.

Reply:
Yep, I though about this, but haven't added initially)

Now committed change to `TestStreamsExtractStreamMetricsExcludeConsumer` test to check for metrics that shouldn't be collected. PTAL.

P.S. Probably not ideal implementation, but works fine.